docs/source/sysadmin_guide/configuring_sawtooth/poet_sgx_enclave_configuration_file.rst

```diff
@@ -49,3 +49,15 @@ The ``poet_enclave_sgx.toml`` configuration file has the following options:
   Identifies the PEM-encoded certificate file that was submitted to Intel in
   order to obtain a SPID. Default: none. Specify the full path to the
   certificate file.
+  This pem file can be created from ``cert.crt`` and ``cert.key`` files
+  with command:
```

Comment:
Please change to "with this command:"  (Or you could say "with the following command:".)

Reply:
How does "with this command " differ vs "with command". They're expressing the same meaning and intent, in this context.

Reply:
It's a grammar thing.  The phrase "with command" needs a definite article ("the") or a determiner ("this").  I should have warned you that I'm a technical writer, not a developer, so I nitpick about grammar and wording.

But you're right - the meaning is clear, so I'll approve this PR. I can fix the grammar later, when I edit the whole Sawtooth doc.

Reply:
In this context, I chose to delete the determiner, as a choice of style. But, I'll just change that.